game-app/game-core/src/main/java/games/strategy/triplea/ui/panel/move/MovePanel.java

```diff
@@ -233,21 +223,13 @@ private void selectUnitsToMove(
           } else { // add one
             // best candidate unit for route is chosen dynamically later
             // check for alt key - add 1/10 of total units (useful for splitting large armies)
-            final List<Unit> unitsToMove = CollectionUtils.getMatches(units, unitsToMoveMatch);
-            unitsToMove.sort(UnitComparator.getHighestToLowestMovementComparator());
-
             final int maxCount = mouseDetails.isAltDown() ? deselectNumber : 1;
```

Comment:
Nit:
`deselectNumber` should ideally be `DESELECT_NUMBER` (and maybe even a better name at all?)

I haven't checked if this constant is used anywhere else, but in case it isn't maybe there's no need to give it a name at all and just inline the value and adjust the comment accordingly (which is also kinda confusing to me).

Kinda out of scope for this PR, but if it isn't a major inconvenience for you I'd appreciate the change nevertheless.

Reply:
Done, changed to MULTI_SELECT_NUMBER. Also updated some comments.